src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -209,6 +209,17 @@ public void testInstructorSubmitAndResultsPage() throws Exception {
     }
 
     @Test
+    public void testInstructorResultsPageForRankRecipientQuestion() throws Exception {
+        ______TS("Rank recipient self excluding instructor results : question");
+
+        InstructorFeedbackResultsPage instructorResultsPage =
+                loginToInstructorFeedbackResultsPageWithViewType("instructor1", "student", false, null);
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-3", "ajax_auto");
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-9", "ajax_auto");
+        instructorResultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageRankRecipientExcludingSelfView.html");
+    }
```

Comment:
Why do we need a new test? An existing test did not cover the question type in concern?